droidlet/tools/hitl/nsp_retrain/ping_cuberite.py

```diff
@@ -0,0 +1,33 @@
+"""
+Copyright (c) Facebook, Inc. and its affiliates.
+"""
+
+import socket
```

Comment:
This is a copy of the original one so that the utils can be used standalone without importing any droidlet modules. I just deleted this file and import it from tools.servermgr.ping_cuberite